### ISSUE TITLE
It should be possible to unselect a column with -c

### ISSUE THREAD

**Issue by chrisnas** [Maoni0/realmon#27]
When -c is used, it is possible to add a column with SPACE but it has no effect on an already selected column.

**Comment by MokoSan** [comment]
This one is a weird one.. I have had to keep the space bar pressed longer than the unchecked one to get the default columns removed. Investigating to see if there is an easy way to fix this.

**Comment by chrisnas** [comment]
You're right: I did not try to keep the SPACE key pressed for a long time.
Thanks for the investigation  :^)
